Fix broken submodule

taoky opened the pull request:
The old submodule `libc-database @ 4ebb57b` is broken. Updated to the latest commit in this PR.

taoky committed:
fix broken submodule

Byzero512 commented:
[submodule "libc-database"]
-    path = libc-database
-    url = https://github.com/lieanu/libc-database.git
+	path = libc-database
+	url = https://github.com/lieanu/libc-database.git
-----------------------------------------------------------------------
so the change have any difference compared with before?

taoky commented:
Well, if you click on the [libc-database @ 4ebb57b](https://github.com/lieanu/libc-database/tree/4ebb57bbda779f5979347934e52e7ca27bc9153f) on [https://github.com/lieanu/LibcSearcher](https://github.com/lieanu/LibcSearcher), you can only get a 404 error page. And if you clone this repo and run `git submodule update --init --recursive
`, it will fail with the error `Server does not allow request for unadvertised object`. This PR is to solve this misconfiguration.

However, I didn't find out how to modify the commit id directly, so I deleted this submodule and added it again to fix this problem. And... `.gitmodules` is modified.

The real change that takes effect is "Submodule libc-database updated from 4ebb57 to 309334".

Byzero512 commented:
but there is another question:
when in libc-database:
use ./get: an erroe happen
-----------------------------------------------------
Getting package location for ubuntu-vivid-i386
Failed to get package version
Getting 
  -> Location: ubuntu-vivid-i386-libc6
  -> ID: 
  -> Downloading package
Failed to download package from ubuntu-vivid-i386-libc6
--------------------------------------------------------------------------
and when git clone libc-database directly, and then use ./get, there isn't the error.

taoky commented:
If you go to see the [Ubuntu support timeline](https://en.wikipedia.org/wiki/Ubuntu_version_history#Version_timeline), you will find that Canonical is still supporting 14.04 (Trusty), because it is a LTS (Long Time Support) version. However, 15.04 (Vivid) is no longer supported. If you still wanna download the `libc` of old versions, go to [old-releases.ubuntu.com](http://old-releases.ubuntu.com) instead of [security.ubuntu.com](http://security.ubuntu.com).

Well, it seems that the author of this repo forked `libc-database` himself (or herself?), and modified some code about 2 years ago. So I guess that his own forked repo is out-of-date for this change. The original `libc-database` has modified the `get` shell script and commented the command of downloading `libc` of some old Ubuntu versions (including vivid).

So, maybe, the best solution is to modify the `get` script and comment the out-of-date commands, or directly change the submodule url to the original `libc-database` instead of the old repo forked by the author.
